core/src/main/java/io/questdb/cairo/pool/WriterPool.java

```diff
@@ -418,7 +418,7 @@ private TableWriter getWriterEntry(
             String lockReason,
```

Comment:
nit:
```suggestion
            @NotNull String lockReason,
```

Reply:
Done in 7adcc9e1dd6516e6450a730329ddf0e29003e5bc